collection_app_ahardman_stylesheet.css

```diff
@@ -18,6 +17,12 @@ body {                 	/* removes default padding and margin around the ouside
     width: device-width;
 }
 
+button:focus {
```

Comment:
missing space after :

Reply:
it's the pseudo-element for when the button is selected rather than a property: value pair